src/components/highlight/index.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import { H1 } from '../heading';
+import Highlight from './index.tsx';
+
+<Meta
+  title="Components/Highlight"
```

Comment:
Do you think is possible to add the story in a category? maybe in `components/utils`?